server/runtime_nakama_module.go

```diff
@@ -1046,18 +1047,21 @@ func (n *NakamaModule) notificationsSendId(l *lua.LState) int {
 				}
 				notification.Subject = v.String()
 			case "Content":
-				if v.Type() != lua.LTString {
+				if v.Type() != lua.LTTable {
 					conversionError = true
-					l.ArgError(1, "expects Content to be string")
+					l.ArgError(1, "expects Content to be a table")
 					return
 				}
-				j := []byte(v.String())
-				maybeJson := make(map[string]interface{})
-				if jsonErr := json.Unmarshal(j, &maybeJson); jsonErr != nil {
-					l.ArgError(1, "expects Content to be a valid JSON")
+
+				contentMap := ConvertLuaTable(v.(*lua.LTable))
+				contentBytes, err := json.Marshal(contentMap)
```

Comment:
There is no conversion happening anywhere else, because sending Notifications can only happen via Lua.